desktop/apps/editorial_features/components/venice_2017/templates/cover.jade

```diff
@@ -0,0 +1,12 @@
+- title = 'Inside the Biennale'
+- subtitle = 'CHRISTIAN MARCLAY'
+- length = '3:54'
+- background = 'https://artsy-vanity-files-production.s3.amazonaws.com/images/unspecified2.jpg'
+
+.venice-overlay(style="background-image:url(#{background})")
```

Comment:
Good job keeping namespacing here—as long as we remember to use something different for 2018's `venice` project :trollface: